python/w2dyn_cthyb/solver.py

```diff
@@ -256,7 +259,19 @@ def solve(self, **params_kw):
             print('cfg["QMC"]["flavourchange_moves"] ',  cfg["QMC"]["flavourchange_moves"])
             print('cfg["QMC"]["statesampling"] ', cfg["QMC"]["statesampling"])
 
+        ### Manually supplied cfg paramters
```

Comment:
This is a compromise between "ease of use" and "flexibility". The interface makes it possible to do simple calculations with `W2Dynamics` using the same interface as `TRIQS/cthyb` and this is a feature I think we would like to keep.

However, for power users that would like to be able to run the solver and supply all parameters in the `W2Dynamics` internal format, this can be done using `cfg_qmc`.

The match is not perfect since the solver builds up its own `cfg_qmc` dictionary with settings based in the  `TRIQS/cthyb` like interface.

This is a compromise between the two.